tests/build_/test_user_params.py

```diff
@@ -392,11 +392,11 @@ def test_validate_missing_required(self):
         with pytest.raises(OsbsValidationException):
             spec.validate()
 
-    @pytest.mark.parametrize('origin_nvr, final_name', [
-        ('test-1-123', 'test-source'),
-        ('test-dashed-nvr-1-123', 'test-dashed-nvr-source'),
+    @pytest.mark.parametrize('origin_nvr, final_name, origin_id', [
+        ('test-1-123', 'test-source', 12345),
+        ('test-dashed-nvr-1-123', 'test-dashed-nvr-source', 12345),
```

Comment:
Let's also have a test which does not pass in an ID.

Reply:
ok